File: src/bot/character/keywords.js
export const KEYWORDS = {
  angry: ['hate', 'angry', 'mad', 'furious'],
  happy: ['love', 'great', 'nice', 'awesome'],
  technical: ['cpu', 'temperature', 'system', 'load'],
};

export function extractKeywords(text) {
  const found = [];
  const lower = text.toLowerCase();

  for (const [k, words] of Object.entries(KEYWORDS)) {
    if (words.some(w => lower.includes(w))) {
      found.push(k);
    }
  }

  return found;
}

File: src/cognition/layer/keywords.py

KEYWORDS = {
    "angry": ["hate", "angry", "mad", "furious"],
    "happy": ["love", "great", "nice", "awesome"],
    "technical": ["cpu", "temperature", "system", "load"],
}

def extract_keywords(text):
    found = []
    lower = text.lower()
    for k, words in KEYWORDS.items():
        if any(w in lower for w in words):
            found.append(k)
    return found
